src/parsers/manifest/dash/common/parse_representations.ts

```diff
@@ -120,19 +122,19 @@ export default function parseRepresentations(
   const parsedRepresentations : IParsedRepresentation[] = [];
   for (const representation of representationsIR) {
     // Compute Representation ID
-    let representationID = representation.attributes.id != null ?
+    let representationID = !isNullOrUndefined(representation.attributes.id) ?
```

Comment:
I'm under the impression than none of those attributes can be set to `null`, in which case the proper way of fixing that double equal to make it more explicit would be to just check against `undefined` instead